### PR TITLE
Show tooltip timestamp on time hover across pages

### LINKED ISSUE #3631: Show tooltip timestamp on time hover across pages
##### What would you like to be added
We should show tooltip timestamps on hover of absolute and relative time displayed across the components to give specific time information to the users.

For example, dates as shown below are not very helpful, because I might want to know exactly when the events occured.

screenshot from 2019-03-04 18-18-49

##### Why is this needed
To improve the user experience of users by giving them more information about dates and times across the dashboard.

##### Comments
In general, tooltips can look like below, with the dotted underline added as a visual cue for users to know they can hover:

screenshot from 2019-03-04 18-15-43
(credit: [this article](https://uxmovement.com/content/absolute-vs-relative-timestamps-when-to-use-which/))

To proceed, you might need to check places where times/dates are shown, e.g. events, conditions etc, and then investigate the use of [Material Tooltips](https://material.angular.io/components/tooltip/overview) for the tooltips.

/good-first-issue

### PR DESCRIPTION
**Pull request by feloy**
Fixes #3631

### CODE REVIEW

**Comment by jeefy** [comment]
This looks :100: 

Small suggestion: Could you possibly the formatted times have some style that lets you know you _can_ hover? Even as simple as a dotted underline would be great. 

/assign

**Comment by jeefy** [comment]
Changes look good.

/approve

**Comment by maciaszczykm** [comment]
Please rebase/merge master to fix the issues with the CI job. Sorry for the inconvenience.

File: src/app/frontend/common/components/date/component.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+ */
+@Component({selector: 'kd-date', templateUrl: './template.html', styleUrls: ['./style.scss']})
+export class DateComponent {
+  @Input() date: string;
+  @Input() format = 'medium';
+
+  _relative: boolean;
+  @Input('relative')
+  set relative(v: boolean) {
+    this._relative = v !== undefined;
```
Comment (maciaszczykm) [inline src/app/frontend/common/components/date/component.ts]: What if somebody would use `[relative]="false"`?

Comment (feloy) [reply src/app/frontend/common/components/date/component.ts]: You are right, I'll use this case

**Comment by jeefy** [comment]
Rebase looks good. @maciaszczykm PTAL

**Pull request merged**